packages/odyssey-react-mui/src/EmptyState.tsx

```diff
@@ -28,37 +28,51 @@ const EmptyContainer = styled("div", {
   padding: odysseyDesignTokens.Spacing5,
   textAlign: "center",
   width: "100%",
+  alignItems: "center",
 }));
 
-export type DataTableEmptyStateProps = {
+export type EmptyStateProps = {
+  /**
+   * Main heading of the empty state
+   */
   heading: string;
+  /**
+   * A descriptive text explaining more context as to why we don't have data.
+   */
   text: string;
-  primaryButton?: ReactNode;
-  secondaryButton?: ReactNode;
+  /**
+   * Primary call to action
+   */
+  primaryCallToActionComponent?: ReactNode;
+  /**
+   * Secondary call to action
+   */
+  secondaryCallToActionComponent?: ReactNode;
 };
 
-const DataTableEmptyState = ({
+const EmptyState = ({
   heading,
   text,
-  primaryButton,
-  secondaryButton,
-}: DataTableEmptyStateProps) => {
+  primaryCallToActionComponent,
```

Comment:
Should these props be `PrimaryCallToActionComponent`and `SecondaryCallToActionComponent `? We've followed that pattern in other places where we know the prop will be a `ReactNode`. https://github.com/okta/odyssey/blob/main/packages/odyssey-react-mui/src/Field.tsx#L88 @KevinGhadyani-Okta any thoughts here?